src/gui_common/charts/line/LineChart.cs

```diff
@@ -263,6 +267,23 @@ public override void _Ready()
         UpdateAxesName();
     }
 
+    public override void _EnterTree()
+    {
+        base._EnterTree();
+
+        ReRegisterToolTips();
```

Comment:
This method is named like this because it is actually not capable of creating new tooltips, it can only reapply existing registrations that are on hold.